packages/rancher-logging/generated-changes/patch/values.yaml.patch

```diff
@@ -35,13 +26,19 @@
  
  affinity: {}
  
-@@ -76,4 +81,69 @@
- monitoring:
-   # Create a Prometheus Operator ServiceMonitor object
-   serviceMonitor:
--    enabled: false
-\ No newline at end of file
-+    enabled: false
+@@ -51,7 +56,7 @@
+     labels: {}
+ 
+ rbac:
+-  enabled: true
++  enabled: false
```

Comment:
Why are we choosing to disable this?

Reply:
It did not exist prior and we have our own overlay. However, I will double check to see what the enabled rbac entails

Reply:
Nevermind. I was off and nothing changed. Fixed.